docs/explanations/db_authz.rst

```diff
@@ -166,7 +166,7 @@ Another option is to define the function with the :code:`SECURITY DEFINER` optio
   -- login as a user wich has privileges on the private schemas
 
   -- create a sample function
-  create or replace function login(email text, pass text) returns jwt_token as $$
+  create or replace function login(email text, pass text, out token text) as $$
```

Comment:
Hm, why not use `RETURNS TABLE (token text)`. Because it's longer?

Not adamantly opposed, just that I've tried to avoid the OUT params before and use RETURNS consistently.